orttraining/orttraining/training_ops/cpu/torch/torch_custom_function_kernel_base.cc

```diff
@@ -30,11 +30,11 @@ void PythonOpBase::Init(const OpKernelInfo& info) {
   ORT_THROW_IF_ERROR(info.GetAttr("name", &name_));
   inplace_ = info.GetAttrOrDefault("inplace", static_cast<int64_t>(0));
   is_training_mode_ = static_cast<bool>(info.GetAttrOrDefault("training_mode", static_cast<int64_t>(0)));
-  ORT_THROW_IF_ERROR(info.GetAttr("call_convention", &call_convention_));
+  ORT_THROW_IF_ERROR(info.GetAttr("input_convention", &input_convention));
+  input_requires_grads_ = info.GetAttrsOrDefault("input_requires_grads", std::vector<int64_t>());
```

Comment:
Should input_requires_grads always be defined?  If so we should omit the default and fail if it is missing (avoiding an extra allocation).

Reply:
Good suggestion! input_requires_grads  should always be defined, we should fail otherwise.

Reply:
Fixed all